rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -297,6 +297,8 @@ cloud_resource_warning_vcpus: 80.0
 cloud_resource_critical_vcpus: 90.0
 cloud_resource_warning_disk_space: 80.0
 cloud_resource_critical_disk_space: 90.0
+cloud_resource_cpu_allocation_ratio: 2
+cloud_resource_mem_allocation_ratio: 1
```

Comment:
Yeah as discussed in the issue, this is not ideal but other than referring to vars from another role (which I think is what you're suggesting) I couldn't think of another way.